incubator/hnc/hack/prow-e2e/Dockerfile

```diff
@@ -29,6 +35,3 @@ RUN go get sigs.k8s.io/kind@v0.9.0
 # Add the actual script and set it as the default command for this image.
 COPY run-e2e-tests.sh run-e2e-tests.sh
 RUN chmod +x ./run-e2e-tests.sh
-
-# This needs to be re-specified in Prow
-CMD ["wrapper.sh", "./run-e2e-tests.sh"]
```

Comment:
Do we not want any entrypoint/cmd?

Reply:
The base image already has the entrypoint set to `wrapper.sh`, and IMO there's no point setting a default command of `run-e2e-tests.sh` since Prow requires that you override it anyway.